python/pyhail/dataset.py

```diff
@@ -958,10 +1443,98 @@ def mendel_errors(self, output, fam):
         """Find Mendel errors; count per variant, individual and nuclear
         family.
 
+        **Implementation Details**
+
+        This method finds all violations of Mendelian inheritance in each (dad,
+        mom, kid) trio of samples.
+
+        The following expression,
+
+        >>> vds.mendel_errors('genomes', 'trios.fam')
+
+        outputs four TSV files according to the `Plink mendel
+        formats <https://www.cog-genomics.org/plink2/formats#mendel>`_:
+
+        - ``genomes.mendel`` -- all mendel errors: FID KID CHR SNP CODE ERROR
+        - ``genomes.fmendel`` -- error count per nuclear family: FID PAT MAT CHLD N NSNP
+        - ``genomes.imendel`` -- error count per individual: FID IID N NSNP
+        - ``genomes.lmendel`` -- error count per variant: CHR SNP N
+
+        FID, KID, PAT, MAT, and IID refer to family, kid, dad, mom, and
+        individual ID, respectively, with missing values set to ``0``.
+
+        SNP denotes the variant identifier ``chr:pos:ref:alt``.
+
+        N counts all errors, while NSNP only counts SNP errors (NSNP is not in Plink).
+
+        CHLD is the number of children in a nuclear family.
+
+        The CODE of each Mendel error is determined by the table below,
+        extending the `Plink
+        classification <https://www.cog-genomics.org/plink2/basic_stats#mendel>`_.
+
+        Those individuals implicated by each code are in bold.
+
+        The copy state of a locus with respect to a trio is defined as follows,
+        where PAR is the pseudo-autosomal region (PAR).
+
+        - HemiX -- in non-PAR of X, male child
+        - HemiY -- in non-PAR of Y, male child
+        - Auto -- otherwise (in autosome or PAR, or female child)
+
+        Any refers to :math:`\{ HomRef, Het, HomVar, NoCall \}` and ! denotes complement in this set.
+
+        +------+------------+------------+----------+------------+
+        | Code | Dad        | Mom        | Kid      | Copy State |
+        +======+============+============+==========+============+
```

Comment:
Did you intend for the genotypes to be italicized? I think it would look better if they weren't. Up to you.